src/metabase/search/engines/appdb/specialization.clj

```diff
@@ -0,0 +1,35 @@
+(ns metabase.search.engines.appdb.specialization
```

Comment:
should this be `metabase.search.engines.appdb.specialization/api`?

Reply:
Yeah I guess that's a bit more consistent

Reply:
It feels strange calling it impl, because it's only a small part of the implementation.

Reply:
It feels strange calling it impl, because it's only a small part of the implementation.